src/main/scala/io/shiftleft/js2cpg/io/FileDefaults.scala

```diff
@@ -44,7 +44,8 @@ object FileDefaults {
     ".*webpack\\..*\\.js".r,
     ".*vue\\.config\\.js".r,
     ".*babel\\.config\\.js".r,
-    ".*chunk-vendors\\.js".r,
+    ".*chunk-vendors.*\\.js".r, // commonly found in webpack / vue.js projects
+    ".*app~.*\\.js".r, // commonly found in webpack / vue.js projects
```

Comment:
Would this ignore app.js?

Reply:
`~` is not a special regex character. So it would only ignore `app~something.js`